drivers/serial/uart_stm32.c

```diff
@@ -654,6 +675,29 @@ static int uart_stm32_err_check(const struct device *dev)
 	return err;
 }
 
+#ifdef CONFIG_UART_WIDE_DATA
```

Comment:
Why isn't this section not with other poll functions ?

Reply:
Moved

Reply:
My bad, I missed a commit somehow